pyro/infer/trace_elbo.py

```diff
@@ -26,7 +26,22 @@ def _compute_log_r(model_trace, guide_trace):
 
 class Trace_ELBO(ELBO):
     """
-    A trace implementation of ELBO-based SVI
+    A trace implementation of ELBO-based SVI. The estimator is constructed
+    along the lines of references [1] and [2]. There are no restrictions on the
+    dependency structure of the model or the guide. The gradient estimator includes
+    partial Rao-Blackwellization for reducing the variance of the estimator when
+    non-reparameterizable random variables are present. The Rao-Blackwellization is
+    partial in that it only uses conditional independence information that is marked
+    by :class:`~pyro.iarange` contexts. For more fine-grained Rao-Blackwellization,
+    see :class:`TraceGraph_ELBO`.
```

Comment:
Thanks for adding nice links! I think you'll need to be explicit about the module:
```
:class:`~pyro.infer.tracegraph_elbo.TraceGraph_ELBO`
```